src/components/matches/matches.component.js

```diff
@@ -37,22 +37,28 @@ const Matches = (props) => {
   }, [location.search]);
 
   return (
-    <div id='matches-page'>
+    <div id="matches-page">
       <div className="spacer_l"></div>
       <div class="spacer_xl"></div>
```

Comment:
just noticed this should be className, could you add a quick fix to it?

Reply:
just pushed up the className fix!

Reply:
Meant to add the comment on line 42, not 40. line 40 should still be `id` but line 42 should be `className` instead of `class` sorry for the miscommunication

Reply:
Oh no worries, just fixed it. Good catch.